sources/sources.go

```diff
@@ -70,13 +70,15 @@ func (s *Sources) Sync(p *ui.UI, force bool) error {
 
 // ForURIs returns Source instances for given uri strings
 func ForURIs(b *ui.UI, dir, env string, uris []string) (*Sources, error) {
-	sources := make([]Source, len(uris))
-	for i, uri := range uris {
+	var sources []Source
```

Comment:
`make([]Source, 0, len(uris))` maybe? the total length of sources can't exceed length of uris, right?

Reply:
True, will change 👍